dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -115,11 +165,15 @@ def update_profile(self, df_series):
         """
         df_series = df_series.apply(str)
         for column_profile in self._profiles:
-            self._profiles[column_profile].update(df_series)
+            self._profiles[column_profile].update(df_series)            
+        return self
```

Comment:
why do the compiler updates need self as well? is that for consistency?

Reply:
Next stage, but yes consistency. Goal was all updates return (no harm in doing so)